### PR TITLE
[Chronicle-246] Google Chronicle Detection & User Alert

### PR DESCRIPTION
**Pull request by crestdatasystems**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
A new feature of Detection Alert is introduced

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by ChanochShayner — COMMENTED** [review]
Great job!!
I added some small comments,
Feel free to ask any question you have.

File: Packs/GoogleChronicleBackstory/ReleaseNotes/2_0_0.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+
+#### Classifiers
+##### New: Chronicle - Incoming Mapper
+- Maps incoming Chronicle incident fields. (Available from Cortex XSOAR 5.0.0).
```
[the remark this answers is not in the document]
Comment (crestdatasystems) [reply Packs/GoogleChronicleBackstory/ReleaseNotes/2_0_0.md]: Yes, it was a typo, we have updated it to 6.0.0.

File: Packs/GoogleChronicleBackstory/ReleaseNotes/2_0_0.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+##### New: Google Chronicle Backstory
+-  (Available from Cortex XSOAR 5.0.0).
+##### New: Chronicle - Classifier
+- Classifies Chronicle incidents (Available from Cortex XSOAR 5.0.0).
```
[the remark this answers is not in the document]
Comment (crestdatasystems) [reply Packs/GoogleChronicleBackstory/ReleaseNotes/2_0_0.md]: Yes, it was a typo, we have updated it to 6.0.0.

File: Packs/GoogleChronicleBackstory/ReleaseNotes/2_0_0.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+- **Chronicle Detection Window End Time**
+- **Chronicle Detection ID**
+- **Chronicle Rule Name**
+- **Chronicle Detection Time**
+- **Chronicle Last Seen**
+- **Chronicle Rule ID**
+- **Chronicle First Seen**
+- **Chronicle Source Product**
+
+#### Incident Types
```
[the remark this answers is not in the document]
Comment (crestdatasystems) [reply Packs/GoogleChronicleBackstory/ReleaseNotes/2_0_0.md:31]: Yes, we have added them in the latest commit.

**Comment by ChanochShayner** [comment]
Demo was held today with @crestdatasystems,
It was great!!
There are small changes that need to be made, and after that, the PR is ready for merge.

**Review by guykeller — CHANGES_REQUESTED** [review]

File: Packs/GoogleChronicleBackstory/.secrets-ignore [hunk 4]
```diff
@@ -15,4 +15,9 @@ https://demo.com
...
 text-align:center;'><h1>1<br/>Good</h1></div>"
 text-align:center;'><h1>No
-Isolated</h1></div>"\
\ No newline at end of file
+Isolated</h1></div>"\
+https://dummy-chronicle
+'bccf9ae7dbfdc1fcaea98fe4043fa3f20f5c4f38a71bad062c8b2d849d79bed8']})
+test_gcb_list_user_alert_with_no_arg_supplied_success(mocker,
+test_gcb_list_user_alert_when_no_alert_found(mocker,
+'21a03d1fa2ce7e342534447e947a94b9f9f0ccfc57e96e86ca56a0074b646852',
```
[the remark this answers is not in the document]
Comment (crestdatasystems) [reply Packs/GoogleChronicleBackstory/.secrets-ignore:23]: yes @guykeller, these hashes are used in pytest mock.

**Review by guykeller — APPROVED** [review]

**Pull request merged**